CHANGELOG.rst

```diff
@@ -14,13 +14,18 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[2.4.8] - 2020-10-19
+~~~~~~~~~~~~~~~~~~~~
+
+* Updated error messaging for inactive accounts
```

Comment:
A more accurate changelog statement would be
` Create a separate error message for inactive users. Refined the existing error message to only for network error or service disruption.`